### ISSUE TITLE
Restart appimagelauncherd for every user after updates

### ISSUE THREAD

**Issue by Igetin** [TheAssassin/AppImageLauncher#143]
AppImageLauncher fails to execute the remove command after right-clicking an application in XFCE’s Whisker Menu and selecting “Remove AppImage from system”. The ` .desktop` files generated by AppImageLauncher in `.local/share/applications` seem to be missing `/usr/` in the `Exec` key. For example:

```
Exec=lib/x86_64-linux-gnu/appimagelauncher/remove /opt/appimages/app_name.AppImage
```

Running manually `/usr/lib/x86_64-linux-gnu/appimagelauncher/remove /path/to/appimage` works properly, as expected.

**Comment by TheAssassin** [comment]
What version are you running?

**Comment by Igetin** [comment]
I’m running version 1.1.1.

**Comment by TheAssassin** [comment]
The new release 1.2.0 is being built right now, will be uploaded soon-ish. Please try that then, it should fix the issue. Please run the app once so that the desktop file gets rewritten.

**Comment by TheAssassin** [comment]
Have you had a chance to try 1.2.0, @Igetin? I have tested a bit, and I cannot reproduce the issue any more.

**Comment by Igetin** [comment]
I uninstalled 1.1.1 and then installed 1.2.2. I tried integrating a couple of AppImages, and the generated desktop files are still missing `/usr/` from the start of the `Exec` path. I’m running Ubuntu 18.04 and using the 64-bit Bionic build.

**Comment by TheAssassin** [comment]
Did you reboot in the meantime? Or at least restarted `appimagelauncherd`? It's nearly impossible to restart user services automatically.

**Comment by Igetin** [comment]
Apparently I was a bit too hasty, the feature seems to work after a reboot.

**Comment by TheAssassin** [comment]
So the issue seems to be that appimagelauncherd had rewritten the desktop files with the old style.

I'll brainstorm on how to reliably auto-restart the user service on changes.

**Comment by azubieta** [comment]
Just a quote here. You cannot restart it for every user as it's not being installed for every user.

**Comment by TheAssassin** [comment]
Fixed by #207.
